zypp/misc/CheckAccessDeleted.h

```diff
@@ -82,20 +85,23 @@ namespace zypp
        */
       size_type check( bool verbose_r = false );
 
-      bool empty() const		{ return _data.empty(); }
-      size_type size() const		{ return _data.size(); }
-      const_iterator begin() const	{ return _data.begin(); }
-      const_iterator end() const	{ return _data.end(); }
+      size_type check( const Pathname &lsofOutput, bool verbose_r = false );
+
+      bool empty() const;
+      size_type size() const;
+      const_iterator begin() const;
+      const_iterator end() const;
+
+      void setDebugOutputFile (const Pathname &filename);
```

Comment:
`/** ... */` a few words about the method.  And please add a short paragraph to the class description about the added feature.